qiskit_ibm_provider/transpiler/passes/scheduling/__init__.py

```diff
@@ -98,6 +104,243 @@
+able to do this in *parallel* on disjoint qubits that satisfy this condition.
+
+For example, the conditional gates below are performed in parallel with lower latency
+as the measurements flow directly into the conditional blocks which in turn only apply
+gates to the same measurement qubit.
+
+.. jupyter-execute::
+
+        qc = QuantumCircuit(2, 2)
+        qc.measure(0, 0)
+        qc.measure(1, 1)
+        # Conditional blocks will be performed in parallel in the hardware
+        with qc.if_test((0, 1)):
+            qc.x(0)
+        with qc.if_test((1, 1)):
+            qc.x(1)
+
+        qc.draw(output="mpl")
+
+
+The circuit below will not use the fast-path as the conditional gate is
+on a different qubit than the measurement qubit.
+
+.. jupyter-execute::
+
+        qc = QuantumCircuit(2, 2)
+        qc.measure(0, 0)
+        with qc.if_test((0, 1)):
+            qc.x(1)
+
+        qc.draw(output="mpl")
+
+Similarly, the circuit below contains gates on multiple qubits
+and will not be performed using the fast-path.
+
+.. jupyter-execute::
+
+        qc = QuantumCircuit(2, 2)
+        qc.measure(0, 0)
+        with qc.if_test((0, 1)):
+            qc.x(0)
+            qc.x(1)
+
+        qc.draw(output="mpl")
+
+A fast-path block may contain multiple gates as long as they are on the fast-path qubit.
+If there are multiple fast-path blocks being performed in parallel each block will be
+padded out to the duration of the longest block.
+
+.. jupyter-execute::
+
+        qc = QuantumCircuit(2, 2)
+        qc.measure(0, 0)
+        qc.measure(1, 1)
+        # Conditional blocks will be performed in parallel in the hardware
+        with qc.if_test((0, 1)):
+            qc.x(0)
+            # Will be padded out to a duration of 1600 on the backend.
+        with qc.if_test((1, 1)):
+            qc.delay(1600, 1)
+
+        qc.draw(output="mpl")
+
+This behavior is also applied to the else condition of a fast-path eligible branch.
+
+.. jupyter-execute::
+
+        qc = QuantumCircuit(1, 1)
+        qc.measure(0, 0)
+        # Conditional blocks will be performed in parallel in the hardware
+        with qc.if_test((0, 1)) as else_:
+            qc.x(0)
+            # Will be padded out to a duration of 1600 on the backend.
+        with else_:
+            qc.delay(1600, 0)
+
+        qc.draw(output="mpl")
+
+
+If a single measurement result is used with several conditional blocks, if there is a fast-path
+eligible block it will be applied followed by the non-fast-path blocks which will execute with
+the standard higher latency conditional branch.
+
+.. jupyter-execute::
+
+        qc = QuantumCircuit(2, 2)
+        qc.measure(0, 0)
+        # Conditional blocks will be performed in parallel in the hardware
+        with qc.if_test((0, 1)):
+            # Uses fast-path
+            qc.x(0)
+        with qc.if_test((0, 1)):
+            # Does not use fast-path
+            qc.x(1)
+
+        qc.draw(output="mpl")
```

Comment:
It depends on the measurement result. If `c` is directly a result from predcessor measurements then  `if (c) x $0; if (c) x $1;` will take the same amount of time as `if (c) x $0;` (as it will be done in parallel) as you said with the fast-path so neither is waiting.